feature/account/server/settings/src/main/kotlin/app/k9mail/feature/account/server/settings/ui/incoming/content/IncomingFormItems.kt

```diff
@@ -53,6 +54,9 @@ internal fun LazyListScope.incomingFormItems(
             onTextChange = { onEvent(Event.ServerChanged(it)) },
             label = stringResource(id = R.string.account_server_settings_server_label),
             contentPadding = defaultItemPadding(),
+            keyboardOptions = KeyboardOptions.Default.copy(
+                autoCorrect = false,
+            ),
```

Comment:
`KeyboardOptions` has default values for all parameters. So writing this is equivalent and shorter:

```suggestion
            keyboardOptions = KeyboardOptions(autoCorrect = false),
```

Reply:
Yes, right. I also added it in OutgoingFormItems.kt